sdk/communication/azure-communication-chat/azure/communication/chat/_models.py

```diff
@@ -203,6 +203,8 @@ class ChatThreadProperties(object):
     :vartype created_on: ~datetime.datetime
     :ivar created_by: the chat thread owner.
     :vartype created_by: CommunicationIdentifier
+    :ivar retention_policy: Data retention policy for auto deletion.
+    :vartype retention_policy: ~azure.communication.chat.models.ChatRetentionPolicy
```

Comment:
Is it possible that the service will return `None` here?
```suggestion
    :vartype retention_policy: ~azure.communication.chat.models.ChatRetentionPolicy or None
```